### PR TITLE
feat(unlock-app) - add verifiers page url in dashboard url

### PR DESCRIPTION
**Pull request by kalidiagne**
# Description
This PR adds
- verifiers page URL in dashboard page
- refactor modified files in typescript

NB: 
- Currently "members" and "verifiers" buttons share the same icon, we need a new svg to differentiate 
 Screenshot 2022-06-03 at 10 11 05

# Issues

Fixes #
Refs #

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [x] I have performed a self-review of my own code
- [x] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: unlock-app/src/components/creator/lock/LockIconBar.tsx [hunk 1]
```diff
@@ -59,14 +75,3 @@ LockIconBar.propTypes = {
   withdraw: PropTypes.func.isRequired,
```
Comment (searchableguy) [inline unlock-app/src/components/creator/lock/LockIconBar.tsx:75]: Remove the prop `LockIconBar.propTypes` here. It's not needed if you are using typescript.

File: unlock-app/src/components/creator/lock/LockIconBar.tsx [hunk 2]
```diff
@@ -1,21 +1,31 @@
...
 import CreatorLockStatus from './CreatorLockStatus'
-import Media from '../../../theme/media'
 import withConfig, { ConfigContext } from '../../../utils/withConfig'
-import { TransactionType, TransactionStatus } from '../../../unlockTypes'
 import { AuthenticationContext } from '../../../contexts/AuthenticationContext'
 
-export function LockIconBar({ lock, toggleCode, withdraw, toggleCreditCard }) {
+interface LockIconBarProps {
+  lock: any
+  withdraw: any
```
Comment (searchableguy) [inline unlock-app/src/components/creator/lock/LockIconBar.tsx:12]: Type the withdraw (it should be a function based on the Button.Withdraw component) and lock (from unlockTypes or type package?).

File: unlock-app/src/components/creator/lock/LockIconBar.tsx [hunk 3]
```diff
@@ -28,27 +38,33 @@ export function LockIconBar({ lock, toggleCode, withdraw, toggleCreditCard }) {
...
   return (
-    <StatusBlock>
-      <IconBar>
-        <Buttons.CreditCard as="button" lock={lock} action={toggleCreditCard} />
-        <Buttons.Withdraw as="button" lock={lock} withdraw={withdraw} />
-        <Buttons.Demo
-          as="a"
-          href={`/demo?network=${network}&lock=${lock.address}`}
+    <div
+      className="flex flex-wrap justify-around"
```
Comment (searchableguy) [inline unlock-app/src/components/creator/lock/LockIconBar.tsx:46]: Add `items-center` to class.

File: unlock-app/src/components/creator/lock/LockIconBar.tsx [hunk 4]
```diff
@@ -28,27 +38,33 @@ export function LockIconBar({ lock, toggleCode, withdraw, toggleCreditCard }) {
...
+      style={{
+        maxWidth: '250px',
+      }}
```
Comment (searchableguy) [inline unlock-app/src/components/creator/lock/LockIconBar.tsx:49]: Nit, you can do the same in className using `max-w-[250px]`. I would use a different measurement unit than pixels though.

File: unlock-app/src/components/interface/buttons/lock/Verifiers.tsx [hunk 5]
```diff
@@ -1,13 +1,10 @@
...
 
 const Verifiers = (props: any) => (
   <Button label="Verifiers" {...props}>
-    <UserAdmin
-      style={{ height: '15px', stroke: 'var(--gray)' }}
-      className="text-gray-400"
-    />
+    <Svg.Members name="Verifiers" />
   </Button>
 )
```
Comment (searchableguy) [inline unlock-app/src/components/interface/buttons/lock/Verifiers.tsx:10]: Do we really need one more button? 

I would also use IconButton in the UI library for icon only buttons. It's not fully featured yet but we can modify it from a single place to include stuff like automatic tooltips.

**Review by searchableguy — CHANGES_REQUESTED** [review]
I would pick an icon https://react-icons.github.io/react-icons and use it instead of duplicating them. 

Use tooltip to inform the purpose further.

File: unlock-app/src/components/creator/lock/LockIconBar.tsx (hunk 4 above)
Comment (kalidiagne) [reply unlock-app/src/components/creator/lock/LockIconBar.tsx:49]: oh cool! thk!

**Comment by julien51** [comment]
Let's hold on that until we have implemented the rest of the functionality to check tickets...

**Comment by kalidiagne** [comment]
@julien51 converted as draft

**Comment by kalidiagne** [comment]
update PR with: 

- New styling for button
Screenshot 2022-07-12 at 12 26 19

- New styling for verifiers page
Screenshot 2022-07-12 at 12 23 58

**Comment by julien51** [comment]
Let's talk about this more a bit later. I am not sure this is the best place.

**Comment by julien51** [comment]
@kalidiagne Let's rebase please :)

**Comment by julien51** [comment]
Closing as we need to think more about this in the context of the new dashboard...

**Pull request closed without merging**